server/src/app/GaWorker.ts

```diff
@@ -25,7 +24,13 @@ export class GaWorker {
 
   constructor(fitnessFunction: string) {
     this.fitnessFunction = fitnessFunction;
-    this.rules = [new LeapRule(), new TritoneRule(), new OctiveRule(), new StepwiseRule(), new CounterTenorRule()];
+    this.rules = [
+      new LeapRule(),
+      new TritoneRule(),
+      new OctiveRule(),
```

Comment:
is this supposed to say OctaveRule?

Reply:
lol yes, good catch. Fixed other occurrences of this typo too.